apps/scan/backend/test/helpers/custom_helpers.ts

```diff
@@ -74,6 +86,7 @@ export async function withApp(
     workspace: Workspace;
     mockUsbDrive: MockUsbDrive;
     mockPrinterHandler: MemoryPrinterHandler;
+    mockFujitsuPrinterHandler: MemoryFujitsuPrinterHandler;
```

Comment:
This is a bit awkward - the testing helper will return a print handler for both types of printers, but only one will actually be active depending on what printer the app is using. Could make a sort of discriminated union of the two but then you'd have to assert within each test, instead of just picking your poison.